packages/sonarwhal/src/lib/cli/init.ts

```diff
@@ -180,18 +182,27 @@ export const initSonarwhalrc = async (options: CLIOptions): Promise<boolean> =>
 
     const initialAnswer: inquirer.Answers = await inquirer.prompt(initialQuestion);
 
-    const sonarwhalConfig = initialAnswer.configType === 'predefined' ?
+    const result = initialAnswer.configType === 'predefined' ?
         await extendConfig() :
         await customConfig();
 
-    // No resources installed or no configuration packages available, we have to quit
-    if (!sonarwhalConfig) {
-        return true;
-    }
-
     const filePath: string = path.join(process.cwd(), '.sonarwhalrc');
 
-    await promisify(fs.writeFile)(filePath, JSON.stringify(sonarwhalConfig, null, 4), 'utf8');
+    await promisify(fs.writeFile)(filePath, JSON.stringify(result.config, null, 4), 'utf8');
+
+    if (result.packages) {
+        const isInstalled = getInstalledResources(ResourceType.configuration).includes(getConfigurationName(result.packages[0]));
+
+        if (isInstalled) {
+            return true;
+        }
+
+        try {
+            await installPackages(result.packages);
+        } catch (err) {
+            // Do nothing.
```

Comment:
Maybe print the error? `logger.error(err.message);` ?

Reply:
In fact, `installPackages` shouldn't return an error, this try/catch is not necessary.